batan-server/engine/board.ts

```diff
@@ -158,11 +160,34 @@ export class Board {
         this.addEdge(49, 52);
         this.addEdge(49, 53);
         this.addEdge(50, 53);
+        this.initHarbors();
+    }
+
+    private initHarbors() {
+        this.vertexList[0].harbor = harborType.threeForOne;
+        this.vertexList[3].harbor = harborType.threeForOne;
+        this.vertexList[1].harbor = harborType.wheat;
+        this.vertexList[5].harbor = harborType.wheat;
+        this.vertexList[10].harbor = harborType.ore;
+        this.vertexList[15].harbor = harborType.ore;
+        this.vertexList[26].harbor = harborType.threeForOne;
+        this.vertexList[32].harbor = harborType.threeForOne;
+        this.vertexList[42].harbor = harborType.sheep;
+        this.vertexList[46].harbor = harborType.sheep;
+        this.vertexList[49].harbor = harborType.threeForOne;
+        this.vertexList[52].harbor = harborType.threeForOne;
+        this.vertexList[51].harbor = harborType.threeForOne;
+        this.vertexList[47].harbor = harborType.threeForOne;
+        this.vertexList[38].harbor = harborType.brick;
+        this.vertexList[33].harbor = harborType.brick;
+        this.vertexList[16].harbor = harborType.wood;
+        this.vertexList[11].harbor = harborType.wood;
```

Comment:
Could this be put into a map and iterated through? Like {idx: type}

Reply:
Yes but for sake of time i'm going to leave this as is.